### ISSUE TITLE
Image without alternative text

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#88]
**Let op:** Een oplossing voor dit probleem wordt genoemd in [#84 Lack of accessible name for score block](https://github.com/frameless/developer.overheid.nl/issues/84#issue-2917820052).

**Dit probleem kan op meerdere pagina's aanwezig zijn, waaronder:**
[CBS Cijfers Open Data](https://apis.developer.overheid.nl/apis/cbs-cijfers)

**Probleembeschrijving:**
De voortgangscirkel-afbeelding heeft geen toegankelijke naam, dus de informatie wordt niet doorgegeven aan screenreadergebruikers. Om dit probleem op te lossen, zou de afbeelding een passende alternatieve tekst moeten hebben die dezelfde informatie overbrengt als de visuele weergave, zoals "Cijfer: 4 van de 4" of "Beoordeling: 4/4". De numerieke beoordeling staat al als tekst naast de afbeelding, dus de afbeelding zou eventueel als decoratief gemarkeerd kunnen worden, maar op dit moment is er geen toegankelijke naam voor het scoreblok, zie probleem [#84 Lack of accessible name for score block](https://github.com/frameless/developer.overheid.nl/issues/84#issue-2917820052)

**Referentiecode:**
`<svg viewBox="0 0 1 1">
                <circle cx="0.5" cy="0.5" r="0.44285714285714284" stroke="#9E9E9E" stroke-width="0.11428571428571428" stroke-dasharray="2.782553493179531 2.782553493179531" stroke-dashoffset="0.0" fill-opacity="0" transform="rotate(-90 0.5 0.5)"></circle>
            </svg>`

**Gerelateerde WCAG:**
WCAG 1.1.1 Non-text Content 

**Referentieafbeelding:**

![A screenreader reads an image of a circle as Image](https://github.com/user-attachments/assets/26ae3959-171b-47d9-af7d-6be4a6a1905e)

_**Referentienummer:** A094_
